src/Uno.UWP/UI/Colors.cs

```diff
@@ -35,21 +33,153 @@ public static Color Parse(string colorCode)
 				}
 				else
 				{
-					if (!_colorMap.TryGetValue(colorCode, out var color))
+					uint color = colorCode switch
 					{
-						var property = typeof(Colors).GetProperty(colorCode);
-
-						if (property != null)
-						{
-							_colorMap[colorCode] = color = (Color)property.GetValue(null)!;
-						}
-						else
-						{
-							throw new InvalidOperationException($"The color {colorCode} is unknown");
-						}
-					}
+						nameof(Transparent) => 0x00FFFFFF,
+						nameof(AliceBlue) => 0xFFF0F8FF,
+						nameof(AntiqueWhite) => 0xFFFAEBD7,
+						nameof(Aqua) => 0xFF00FFFF,
+						nameof(Aquamarine) => 0xFF7FFFD4,
+						nameof(Azure) => 0xFFF0FFFF,
+						nameof(Beige) => 0xFFF5F5DC,
+						nameof(Bisque) => 0xFFFFE4C4,
+						nameof(Black) => 0xFF000000,
+						nameof(BlanchedAlmond) => 0xFFFFEBCD,
+						nameof(Blue) => 0xFF0000FF,
+						nameof(BlueViolet) => 0xFF8A2BE2,
+						nameof(Brown) => 0xFFA52A2A,
+						nameof(BurlyWood) => 0xFFDEB887,
+						nameof(CadetBlue) => 0xFF5F9EA0,
+						nameof(Chartreuse) => 0xFF7FFF00,
+						nameof(Chocolate) => 0xFFD2691E,
+						nameof(Coral) => 0xFFFF7F50,
+						nameof(CornflowerBlue) => 0xFF6495ED,
+						nameof(Cornsilk) => 0xFFFFF8DC,
+						nameof(Crimson) => 0xFFDC143C,
+						nameof(Cyan) => 0xFF00FFFF,
+						nameof(DarkBlue) => 0xFF00008B,
+						nameof(DarkCyan) => 0xFF008B8B,
+						nameof(DarkGoldenrod) => 0xFFB8860B,
+						nameof(DarkGray) => 0xFFA9A9A9,
+						nameof(DarkGreen) => 0xFF006400,
+						nameof(DarkKhaki) => 0xFFBDB76B,
+						nameof(DarkMagenta) => 0xFF8B008B,
+						nameof(DarkOliveGreen) => 0xFF556B2F,
+						nameof(DarkOrange) => 0xFFFF8C00,
+						nameof(DarkOrchid) => 0xFF9932CC,
+						nameof(DarkRed) => 0xFF8B0000,
+						nameof(DarkSalmon) => 0xFFE9967A,
+						nameof(DarkSeaGreen) => 0xFF8FBC8B,
```

Comment:
[DarkSeaGreen](https://learn.microsoft.com/en-us/uwp/api/windows.ui.colors.darkseagreen?view=winrt-22621) is #FF8FBC8F.

you might want to codegen those lines with uwp just to be safe:
```cs
var asd = string.Join("\n", typeof(Colors)
	.GetProperties(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static)
	.Where(x => x.PropertyType == typeof(Color))
	.OrderByDescending(x => x.Name == "Transparent")
	.Select(x => $"nameof({x.Name}) => {x.GetValue(null).ToString().Replace("#", "0x")},")
	//.Select(x => $"\"{x.Name.ToLower()}\" => {x.GetValue(null).ToString().Replace("#", "0x")},")
);
```

Reply:
Weird, they were generated from similar code - but from Uno, not UWP itself. I'll double check that, thanks!

Reply:
https://github.com/unoplatform/uno/blob/89bda710bc575be97edf6b87b601b5ed070061c4/src/Uno.UWP/UI/Colors.cs#L456

ok, that's an Uno bug :)
I'll fix it inside this PR

Reply:
I'm really curios how you caught this @Xiaoy312

Reply:
@Youssef1313 2am insanity. just picked a few randomly to sanity check.